src/lib/client.ts

```diff
@@ -104,24 +104,18 @@ function clearStored() {
   }
 }
 
+//TODO should be refactored to not rely on the .init() method to be valid
 export class Client {
-  wsProvider = new WsProvider
-  api: ApiPromise = new ApiPromise
-  farmed = getStoredBlocks()
-  clientData = reactive(emptyData)
-  data = emptyData
-  clearTauriDestroy: event.UnlistenFn = () => { }
-  unsubscribe: event.UnlistenFn = () => { }
-  constructor() {
-    this.clientData.farming.farmed = this.farmed
-  }
-  async init() {
-    this.api = await ApiPromise.create({ provider: this.wsProvider, types: customTypes })
-  }
+  protected wsProvider = new WsProvider
```

Comment:
Just FYI, `protected` should only be used when you intend the property or method to ever be accessible to the class that extends it. Otherwise, it should be `private`.